mots.yaml

```diff
@@ -1,15 +1,11 @@
 repo: mots
 created_at: '2022-02-02T14:48:50.574414'
-updated_at: '2022-04-05T09:21:28.220398'
+updated_at: '2022-04-12T13:35:10.995339'
 export:
   path: ./mots.rst
   format: rst
 people:
-  - &zeid
-    bmo_id: 633708
-    name: Zeid Zabaneh
-    info: '[:zeid]'
-    nick: zeid
+  - &zeid {bmo_id: 633708, name: Zeid Zabaneh, nick: zeid}
```

Comment:
I generally also do not like it, however I think it improves the readability of the people section (e.g. see it in full action [here](https://phabricator.services.mozilla.com/D130508)). Mainly because the most important bits of this dictionary are the anchors/nicks.

Reply:
After removing the `info` field I realized that the amount of text in the original style is not so bad after all. Perhaps in the future we'll remove the `nick` field as well (since it's implied in the anchor) and that would make it even more readable.